content/library/api/connections/connections.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Connections and databases
+slug: /library/api-reference/connections
+---
+
+# Connections and databases
+
+## Setup your connection
+
+<TileContainer>
+<RefCard href="/library/api-reference/connections/st.experimental_connection" size="half">
+
+<Image pure alt="screenshot" src="/images/api/connection.jpg" />
+
+#### Create a connection
+
+Connect to a data source or API
+
+```python
+conn = st.experimental_connection('pets_db', type='sql')
+pet_owners = conn.query('select * from pet_owners')
+st.dataframe(pet_owners)
+```
+
+</RefCard>
+</TileContainer>
+
+## Built-in connections
+
+<TileContainer>
+
+<RefCard href="/library/api-reference/connections/st.connections.sqlconnection" size="half">
+
+<Image pure alt="screenshot" src="/images/api/connections.SQLConnection.jpg" />
+
+#### SQLConnection
+
+A connection to a SQL database using SQLAlchemy.
+
+```python
+conn = st.experimental_connection('sql')
+```
+
+</RefCard>
+
+<RefCard href="/library/api-reference/connections/st.connections.snowparkconnection" size="half">
+
+<Image pure alt="screenshot" src="/images/api/connections.SnowparkConnection.jpg" />
+
+#### SnowparkConnection
+
+A connection to Snowflake Snowpark.
+
+```python
+conn = st.experimental_connection('snowpark')
+```
+
+</RefCard>
+</TileContainer>
+
+## Third-party connections
+
+<TileContainer>
+<RefCard href="/library/api-reference/connections/st.connections.experimentalbaseconnection" size="half">
+
+#### Conection base class
```

Comment:
Typo: should be `Connection base class` (missing `n`)

Reply:
Fixed on 45d40e3.